### DISCUSSION TITLE
Setup Page always report exception: The provided connection string is invalid or server is unreachable.

### DISCUSSION THREAD

**Discussion by hylinux** [OrchardCMS/OrchardCore#13259] [General]
### Describe the bug
when I follow this document:
https://docs.orchardcore.net/en/latest/docs/getting-started/

create my first demo,   when I choose the database to sqlite, everything is ok. but when I tried to choose SQL Server,  what every kind of connection string  was inputed, it always show me exception:

The provided connection string is invalid or server is unreachable.

my connection string is:
Persist Security Info=False;User ID=sa;Password=Password01!@#4;Initial Catalog=TestDB1;Data Source=DESKTOP-8AVVAJV\\TESTDB

the interesting thing is: this connection string was testing success when I use another ASP.net Core with EF Core framework. 

### To Reproduce
1. Go to document: https://docs.orchardcore.net/en/latest/docs/getting-started/ create a demo web application.
2. run the web app. and input the connection string.

please Note: I already create the database which I point out in Connection String.

the Orchard Core version is 1.5.0. 

would you help take a look?

thanks

**Comment by rjpowers10** [comment]
You're most likely running into the breaking changes listed here [https://docs.orchardcore.net/en/latest/docs/releases/1.5.0/#yessql-breaking-changes](https://docs.orchardcore.net/en/latest/docs/releases/1.5.0/#yessql-breaking-changes)

**Comment by hishamco** [comment]
Exactly, the SQL Server connection is now secured, you can ignore it by using `Encrypt=false`

**Comment by hyzx86** [comment]


**Reply by hyzx86** [reply]
Maybe a PR recently fixed the tenant installation screen,



It prevents the connection string from being saved to the installation process.



There was a problem before. After configuring the tenant's connection string and recipe information in the administration panel, there was no problem under normal circumstances.

However, if an error occurs during installation, the recipe and database connection information is displayed on the installation page.


Was it the recent PR that tried to solve the problem, but didn't quite solve it?

**Reply by hyzx86** [reply]
Sorry... I checked the connection string and I made a mistake
